src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media_Animation/Given_ObjectAnimationUsingKeyFrames.cs

```diff
@@ -322,7 +322,7 @@ public void When_FirstFrameTimeIsZero_Then_ItsAppliedSyncOnStart()
 		/// </summary>
 		private IDisposable UseDarkTheme() => ThemeHelper.UseDarkTheme();
 
-#if !NETFX_CORE // Disabled on UWP for now because 17763 doesn't support WinUI 2.x
+#if !NETFX_CORE // Disabled on UWP for now because 18362 doesn't support WinUI 2.x
```

Comment:
Is this still true ?

Reply:
No 👍 !

Reply:
But the tests do use various Uno-specific APIs, so I have to keep them disabled still.